docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,55 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[post-upgrade-reqs-8.0.0]]
+==== Post upgrade requirements
+* Changes to the indicator match rule's default threat indicator path (<<rule-ui-advanced-params, *Indicator prefix override*>>) might require you to update existing rules, or create new ones, after upgrading to {stack} version 8.0.0. Be mindful of the following ({pull}118821[#118821]):
+** If an indicator match rule's default threat indicator path was not defined before upgrading, it will receive the default value `threatintel.indicator` during the upgrade process. This allows the rule to continue using indicator data ingested by {filebeat} version 7.x. If a custom value was defined before the upgrade, the value remains the same.
+** If an existing indicator match rule was configured to use threat indicator indices generated from {filebeat} version 7.x, updating the default threat indicator path to `threat.indicator` after you upgrade to {stack} version 8.0.0 and {agent} or {filebeat} version 8.0.0 configures the rule to use threat indicator indices generated by {agent} or {filebeat} version 8.0.0.
+** Separate rules must be created to query threat intelligence indices created by {filebeat} version 7.x and version 8.0.0 because different default threat indicator path values are required.
```

Comment:
```suggestion
** You must create separate rules to query threat intelligence indices created by {filebeat} versions 7.x and 8.0.0 because each major version requires different default threat indicator path values.
```

Reply:
I'm not sure that it'd be accurate to say that each major version requires a different default value, but I think I get what you're conveying with this suggestion. How about:
 
 You must create separate rules to query threat intelligence indices created by {filebeat} versions 7.x and 8.0.0 because each version requires different default threat indicator path values.